backend/src/graphql/resolver/ContributionResolver.ts

```diff
@@ -174,6 +204,13 @@ export class ContributionResolver {
     contributionToUpdate.contributionStatus = ContributionStatus.PENDING
     dbContribution.save(contributionToUpdate)
 
+    const event = new Event()
+
+    const eventUpdateContribution = new EventContributionUpdate()
+    eventUpdateContribution.userId = user.id
+    eventUpdateContribution.contributionId = contributionId
+    await eventProtocol.writeEvent(event.setEventContributionUpdate(eventUpdateContribution))
```

Comment:
the same here:
in case the attribute amount is mandatory, as requested, then here the initialization is missing

Reply:
true again, solved